src/main.lua

```diff
@@ -828,6 +845,22 @@ Handlers.add(
 			return
 		end
 
+		local services = nil
+        if msg.Tags.Services then
```

Comment:
Nit: some funny tabbing here. Plz run stylua if you can.

Reply:
Ran stylua and fixed the funky tabs and missing commas.